fabtests/benchmarks/dgram_pingpong.c

```diff
@@ -100,7 +100,7 @@ int main(int argc, char **argv)
 			if (!ft_parse_long_opts(op, optarg))
 				continue;
 			ft_parse_benchmark_opts(op, optarg);
-			ft_parseinfo(op, optarg, hints, &opts);
+			ft_parseinfo(op, optarg, &hints, &opts);
```

Comment:
There's a lot of changes in this single patch.  This change should be in a separate patch.  Why was this made to pass in the address of a pointer, rather than the pointer?  That's non-intuitive and suggests the function has a possible side effect.